Userland/Libraries/LibWeb/Painting/BorderRadiusCornerClipper.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022, MacDue <macdue@dueutil.tech>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGfx/AntiAliasingPainter.h>
+#include <LibWeb/Painting/BorderPainting.h>
+
+namespace Web::Painting {
+
+class BorderRadiusCornerClipper {
+public:
+    static ErrorOr<BorderRadiusCornerClipper> try_create(Gfx::IntRect const& border_rect, BorderRadiiData const& border_radii);
```

Comment:
`try_foo` is reserved for when there's already an infallible `foo` method that hasn't been removed yet